rotors_control/src/nodes/lee_position_controller_node.cpp

```diff
@@ -123,24 +107,22 @@ void LeePositionControllerNode::CommandMultiDofJointTrajectoryCallback(
     return;
   }
 
-  mav_msgs::EigenCommandTrajectoryPositionYaw eigen_reference;
-  mav_msgs::eigenCommandTrajectoryPositionYawFromMultiDofJointTrajectoryPointMsg(msg->points.front(),
-                                                                                 &eigen_reference);
+  mav_msgs::EigenTrajectoryPoint eigen_reference;
+  mav_msgs::eigenTrajectoryPointFromMsg(msg->points.front(), &eigen_reference);
   commands_.push_front(eigen_reference);
 
   for (size_t i = 1; i < n_commands; ++i) {
```

Comment:
do we really want a queue for the commands in the lee controller? I think this is dangerous and does not make sense.